src/codewhisperer/explorer/codewhispererNode.ts

```diff
@@ -97,9 +100,12 @@ export class CodeWhispererNode implements RootNode {
                 return [createLearnMore(), createEnableCodeSuggestionsNode()]
             }
         } else {
-            const accessToken = this.getDescription()
-            if (accessToken || AuthUtil.instance.isConnected()) {
+            const isAccessToken = this.getDescription() === 'Access Token'
+            if (isAccessToken || AuthUtil.instance.isConnected()) {
```

Comment:
accessToken is no longer in use. So isAccessToken will always be false.

Reply:
it is ok for it to be false because it's `or`?

Reply:
Yes but since we are deprecating access token, you can remove isAccessToken entirely.

Reply:
it can be in another PR when we remove everything accessToken related?